repos/system_upgrade/common/actors/checkyumpluginsenabled/libraries/checkyumpluginsenabled.py

```diff
@@ -0,0 +1,50 @@
+from leapp import reporting
+from leapp.libraries.common.rhsm import skip_rhsm
+
+REQUIRED_YUM_PLUGINS = ['subscription-manager', 'product-id']
```

Comment:
Make it set right away, there's no need to convert it later 
```suggestion
REQUIRED_YUM_PLUGINS = {'subscription-manager', 'product-id'}
```

Reply:
Implemented in the most recent commit.